types/jest-axe/index.d.ts

```diff
@@ -74,3 +74,9 @@ declare global {
     // axe-core depends on a global Node
     interface Node {}
 }
+
+declare module "@jest/expect" {
```

Comment:
Technically, I think this change to `@jest/expect` only takes effect after you call `expect.extend(toHaveNoViolations)`.
But there's no way to represent that in TS, so either you can have this change here, and people have to remember to do `expect.extend(toHaveNoViolations)`, or, as the original library suggests, you can add these changes to `types/jest-axe/extend-expect.d.ts` to reflect what the library says [here](https://unpkg.com/browse/jest-axe@8.0.0/extend-expect.js).